Cargo.toml

```diff
@@ -30,9 +30,10 @@ tracing = { version = "0.1", default-features = false, features = [ "attributes"
 tracing-subscriber = { version = "0.2" }
 
 [features]
-default = ["parallel", "nova", "hypernova"]
-hypernova=[]
+default = ["parallel", "nova", "hypernova", "protogalaxy"]
 nova=[]
+hypernova=[]
+protogalaxy=[]
```

Comment:
So the idea was that in this way users of the lib can select which ones to use. For example if somebody wants to use the protogalaxy feature, they avoid also loading all the code unrelated to that (eg. Nova).
Does it make sense? or maybe there is another way to achieve this?

Reply:
Generally, Rust doesn't load it by default. It just loads/compiles the necessary code for your program. See [this](https://www.reddit.com/r/rust/comments/dd9w9i/comment/f2f7ouj/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) or other replies. (I haven't been able to locate within the Rust docs this fact..).

So summarizing, you basically can remove the conditional compilation as you will not be having the final artifacts. 
In any case, I just say it because working with conditional compilation it's painful.

Reply:
Oh awesome, thanks for the explanation! Will remove it then.

Reply:
Updated here: https://github.com/privacy-scaling-explorations/folding-schemes/pull/37/commits/233dc9c1fd9a8ddc87c3ca9b4c605770d4eb20a8